### DISCUSSION TITLE
What is flat config equivalents to mdx/recommended

### DISCUSSION THREAD

**Discussion by onemen** [mdx-js/mdx#2491] [Q&A]
What is the equivalents of .eslintrc `"extends":["plugin:mdx/recommended"],` when using eslint 9 with `eslint.config.js`?

**Comment by ChristianMurphy** [comment]
Welcome @onemen! 👋 
Taking a step back your interest seems to be using ESLint MDX with ESLint version 9?
If so see the tracking issue at https://github.com/mdx-js/eslint-mdx/issues/477

/cc @JounQin
